### ISSUE TITLE
[BUG]1006 (connection closed abnormally [internal]), no reason

### ISSUE THREAD

**Issue by electroboyz** [Chia-Network/chia-blockchain#8908]
```
2021-10-22T22:07:37.643 daemon __main__                   : ERROR    Unexpected exception trying to send to websocket: code = 1006 (connection closed abnormally [internal]), no reason Traceback (most recent call last):
  File "asyncio\windows_events.py", line 457, in finish_recv
OSError: [WinError 64] The specified network name is no longer available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
  File "asyncio\proactor_events.py", line 280, in _loop_reading
  File "asyncio\windows_events.py", line 812, in _poll
  File "asyncio\windows_events.py", line 461, in finish_recv
ConnectionResetError: [WinError 64] The specified network name is no longer available

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "Lib\site-packages\chia\daemon\server.py", line 208, in safe_handle
  File "websockets\protocol.py", line 555, in send
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
```

This error prevails even though I uninstall reinstall Chia 1.2.9 any way to fix this?

**Comment by moonlitbugs** [comment]
This looks to be a duplicate of #2810 (and a bunch of other duplicates as well, all closed).
Chiming in a "me too" here, except that Chia is running on Linux and not Windows.

In my case, Chia is running inside a small LXC (Linux container).  Restarting the container "fixed" the issue, at least temporarily.  There is nothing helpful in the Chia debug.log file beyond the endless backtraces.

I do note that chia_full_node is consuming a lot of memory, more than 1.1g resident and 1.5g virtual.  On a small VM, that's dangerously close to putting it into swap space, and/or triggering the "OOM Killer" (Linux's low-memory panic mechanism).

**Comment by waylonwang** [comment]
The official docker container (chia version is 1.2.11) has the same error. This error will cause port 8555 to fail. I cannot view the current full node status through the chia show -sc command. If you delete the database in the mainnet/db directory, restart After the system re-synchronized the full node, there would be no such problem, but after I tried to restart the full node service when I synchronized to a height of 80,000+, the problem occurred again.
